engine/engine/src/engine.cpp

```diff
@@ -590,6 +626,30 @@ namespace dmEngine
 #endif
         }
 
+        // Try loading SSL keys
+        char* paths[2];
+        uint32_t count = 0;
+        char engine_ssl_keys_path[DMPATH_MAX_PATH];
+        if (resources_path)
```

Comment:
This if-statement will always be true, since it's a pointer to a stack variable (it will never be 0)